de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/ASwtBotTestCase.java

```diff
@@ -209,6 +233,53 @@ protected SWTBotTreeItem openEditor(SWTBotTreeItem item) {
 	 * @param diagramEditor Graphiti diagram editor which the tree item is beeing dragged onto
 	 */
 	protected void dragTreeItemToDiagramEditor(SWTBotTreeItem item, SWTBotGefEditor diagramEditor) {
+		item.dragAndDrop(getCanvasForDiagramEditor(diagramEditor));
+		waitForEditingDomainAndUiThread();
+	}
+	
+	/**

```

Comment:
Maybe we can group the dragTreeItemToDIagramEditor methods together and put the getBoundsForEditPart somewhere lower.